extension/flat_tensor/serialize/serialize.py

```diff
@@ -266,7 +266,7 @@ def serialize(
         # Create FlatTensor, which describes of the contents of the file and
         # points to all the data segments. It will be serialized to flatbuffer.
         flat_tensor = FlatTensor(
-            version=0,
+            version=0,  # Keep in sync with c++ version number in serialize.h
```

Comment:
Consider adding a top-level `_SCHEMA_VERSION: int = 0` to raise visibility of this, and to make it map more clearly to the C++ equivalent.